lib/constants.js

```diff
@@ -17,6 +17,9 @@
 'use strict';
 
 module.exports = {
-  /** @const {string} header that is used to identify trace across API calls. */
-  TRACE_CONTEXT_HEADER_NAME: 'X-Cloud-Trace-Context'
+  /** @const {string} header that carries trace context across Google infrastructure. */
+  TRACE_CONTEXT_HEADER_NAME: 'X-Cloud-Trace-Context',
+
+  /** @const {string} header that is used to identify outgoing http made by the agent. */
+  TRACE_IGNORE_HEADER: 'X-Trace-Ignore-Header'
```

Comment:
'Header' in the name of a header is redundant. Since this goes onto the network, maybe a more-descriptive name (e.g. 'X-Cloud-Trace-Agent-Request' might be better).